std_msgs/Header header
goal_manager/GpsGoal[] gps_goals
